### PR TITLE
Feat/style panel

### PR DESCRIPTION
**Pull request by MrCoder**

### CODE REVIEW

File: src/components/DiagramFrame/SeqDiagram/MessageLayer/Block/Statement/InteractionAsync/Interaction-async.vue [hunk 1]
```diff
@@ -7,7 +7,7 @@
     :style="{ width: interactionWidth + 'px', transform: 'translateX(' + translateX + 'px)' }"
   >
     <comment v-if="comment" :comment="comment" />
-    <component v-bind:is="invocation" :content="signature" :rtl="rightToLeft" type="async" :number="number"/>
+    <component v-bind:is="invocation" :context="isSelf ? asyncMessage : undefined" :content="signature" :rtl="rightToLeft" type="async" :number="number"/>
```
Comment (MrCoder) [inline src/components/DiagramFrame/SeqDiagram/MessageLayer/Block/Statement/InteractionAsync/Interaction-async.vue:10]: Why do we pass context only when it is `isSelf`? Was this on purpose?

File: src/store/Store.ts [hunk 2]
```diff
@@ -41,14 +41,17 @@ const Store = ():  StoreOptions<StoreState> => {
...
-        console.log('Element clicked', codeRange);
-      },
       numbering: Boolean(localStorage.getItem(`${location.hostname}-zenuml-numbering`)),
       stickyOffset: 0,
       diagramElement: null,
+      onElementClick: (codeRange: CodeRange) => {
+        console.log('Element clicked', codeRange);
+      },
+      onMessageClick: () => {},
+      onUpdateEditorContent: () => {},
```
Comment (MrCoder) [inline src/store/Store.ts:51]: Same as the above comment on `onUpdateEditorContent`.

**Review by dyon21 — APPROVED** [review]

File: src/store/Store.ts [hunk 3]
```diff
@@ -1,15 +1,22 @@
...
+import { Coordinates } from "../positioning/Coordinates";
+import { CodeRange } from "../parser/CodeRange";
+import { StoreOptions } from "vuex";
 
 let storeInitiationTime: number = 0;
 setTimeout(function () {
   if (!storeInitiationTime) {
-    console.warn('[vue-sequence] Store is a function and is not initiated in 1 second.');
+    console.warn(
+      "[vue-sequence] Store is a function and is not initiated in 1 second.",
```
Comment (MrCoder) [inline src/store/Store.ts:18]: vue-sequence -> @zenuml/core
This can be done in a separate PR.

File: src/utils/String.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
+
+export const getLineHead = (code: string, position: number) => {
```
Comment (MrCoder) [inline src/utils/String.ts:2]: Test should be added to the functions in this file.
Non-blocker.

**Pull request merged**